java/node/src/main/java/org/svarm/node/engine/impl/V1SingleEntryEngine.java

```diff
@@ -198,18 +166,55 @@ public void write(final TenantTable tenantTable, final EntryInfo entryInfo) {
   @Override
   public boolean delete(final TenantTable tenantTable, final String entity) {
     LOGGER.trace("delete({},{})", tenantTable, entity);
-    return sqlEngine.executePreparedTenant(tenantTable,
-        "delete from TENANT_DATA where ID = ?",
-        (ps) -> {
-          try {
-            ps.setString(1, entity);
-            final int rows = ps.executeUpdate();
-            final boolean result = rows > 0;
-            LOGGER.trace("deleted: {}:{}:{}", tenantTable, entity, result);
-            return result;
-          } catch (SQLException e) {
-            throw new IllegalArgumentException("Unable to read data for: " + tenantTable + ":" + entity, e);
-          }
-        });
+
+    int updateCount = Jdbi.create(dataSourceManager.getDataSource(tenantTable))
+        .withHandle(handle -> handle.createUpdate("delete from TENANT_DATA where ID = :id")
+            .bind("id", entity)
+            .execute()
+        );
+
+    final boolean result = updateCount > 0;
+
+    LOGGER.trace("deleted: {}:{}:{}", tenantTable, entity, result);
+    return result;
+  }
+
+  private class EntryInfoResultSetScanner implements ResultSetScanner<Optional<EntryInfo>> {
```

Comment:
I have the feeling this can be simplified. It's with parity of what I had before. (Assuming the tests work) No need to change here. But this probably can be redone in a future  PR.

Reply:
I updated https://github.com/wolpert/svarm/issues/11 to include this simplification.